CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
+* Don't test deprecated hooks.
```

Comment:
This is not really a user-visible change, and it is not necessary to mention it in the change log as it is already in the commit log.